wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -394,6 +399,41 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:restriction>
 			</xs:simpleType>
 			<!--===============================-->
+			<xs:simpleType name="HardwareTypes">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="Camera">
+						<xs:annotation>
+							<xs:documentation>Single sensor camera. Ex. bullet, PTZ or fisheye camera.</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="Multi-sensor">
+						<xs:annotation>
+							<xs:documentation>Multiple sensors device.</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="Encoder">
+						<xs:annotation>
+							<xs:documentation>Analog encoder. Can have 1 or more encoders.</xs:documentation>
+						</xs:annotation>
```

Comment:
suggestion to add Decoder.

Reply:
Do you have a description for what a Decoder is?

Reply:
A video decoder converts an encoded video stream to HDMI output for on screen display.